packages/v4/patternfly-docs/pages/community.js

```diff
@@ -100,7 +100,7 @@ const CommunityPage = () => {
             <SplitItem style={{ marginRight: '12px' }}><ChatIcon /></SplitItem>
             <SplitItem isFilled>
               <Title size="lg" className="ws-title" headingLevel="h3">Chat with us</Title>
-              <a href="//slack.patternfly.org/" target="_blank" rel="noopener noreferrer">Slack</a>
+              <a href="//patternfly.slack.com/" target="_blank" rel="noopener noreferrer">Slack</a>
```

Comment:
Suggest using this invite link which (unlike patternfly.slack.com) will allow new users to sign up to join, otherwise will redirect users to the slack workspace if they're already signed in.

```suggestion
              <a href="//join.slack.com/t/patternfly/shared_invite/zt-1npmqswgk-bF2R1E2rglV8jz5DNTezMQ" target="_blank" rel="noopener noreferrer">Slack</a>
```